Makefile

```diff
@@ -77,7 +77,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	docker build -t ${IMG} . --build-arg TARGETOS=linux --build-arg TARGETARCH=amd64
```

Comment:
We should read OS and ARCH from env variables.

Reply:
This change was unintended and temporary. Will pick these up from envvars anyway.